packages/editor/src/components/provider/index.js

```diff
@@ -134,23 +137,6 @@ class EditorProvider extends Component {
 
 	componentDidMount() {
 		this.props.updateEditorSettings( this.props.settings );
-
-		if ( ! this.props.settings.styles ) {
-			return;
-		}
```

Comment:
Where did this condition go in the new implementation? Was it not needed?

Reply:
Not needed cause we map through the styles but it can be  micro-optim